TEP008.rst

```diff
@@ -0,0 +1,46 @@
+TEP008: Tighter integration of Atomic Data, Plasma and Model
+============================================================
+
+Status
+======
+**Discussion**
+
+Responsible
+===========
+
+@wkerzendorf
+@yeganer
+
+Branches and Pull requests
+==========================
+
+Description
+===========
+
+TARDIS current plasma is structured in a very modular way that allows easy
+testing and quick addition of new plasma modules. We restructured the Plasma
+to this new form in 2015 and implemented the most important parts of the
+framework. Due to time constraints, not all functionality was moved to this new
+structure and there is still considerable amount of legacy code. In addition,
+while the functionality is there we lack documentation that explains the new
+rationale to the collaboration and users.
+
+With this TEP we have multiple aims - ensure that all remaining parts that will
+fit within the new structure are moved. Ensure that we update the documentation
+to better reflect the current state of the code. This will include several
+examples on how to write new plasma modules.
+
+Implementation
```

Comment:
For the implementation i'd suggest adding a short list like this:

- Finish basic documentation ( This is especially important for new GSoC students so they can get an idea what's going on )
- Write 1-2 simple examples for the documentation ( Idea for someone who want's to work on the atom data reader: provide a baseclass for atom data properties )
- Create module for graph handling, then atom_data_properties and plasma_properties will inherit from that (useful for stuff like save, load, to_hdf5 etc.)
- Move the global graph to the model instead of plasma
- Integrate new Atom_data and Plasma into the new model

What do you think?